commands/module-unload.md

```diff
@@ -0,0 +1,13 @@
+Unloads a module.
+
+This command unloads the module specified by `name`. Note that the module's name
+is reported by the `MODULE LIST` command, and may differ from the dynamic
+library's filename.
+
+Known limitations:
+
+*   Modules that register custom data types can not be unloaded.
```

Comment:
IIRC, If you load a module, write some keys with the module data type then restart the server, it will choke trying to restore from RDB because of the missing data types. Should this be in the limitations?

Reply:
I don't think so... you can't load an RDB after unloading a module (unless you use DEBUG). That said, I believe that limitation is buried somewhere in the modules' docs (need to find ref though). In any case, once https://github.com/antirez/redis/pull/5926 is released we'll need to update this.

Reply:
@itamarhaber fair enough. I'm pretty sure that limitation exists if you stop the redis server, change the redis.conf then restart it again. But, I suppose that is a different situation than `MODULE UNLOAD` (which I rarely use)